docs/core/deploying/native-aot/warnings/il3055.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "IL3055: P/invoke method declares a parameter with an abstract delegate"
+description: "Correctness of interop for abstract delegates cannot be guaranteed after native compilation - the marshalling code for the delegate might not be available"
```

Comment:
```suggestion
description: "Learn about warning IL3055: Correctness of interop for abstract delegates can't be guaranteed after native compilation because the marshalling code for the delegate might not be available."
```